packages/block-library/src/post-comment-author-avatar/index.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comment-author-avatar` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comment-author-avatar` block on the server.
+ *
+ * @param array    $attributes Block attributes.
+ * @param string   $content    Block default content.
+ * @param WP_Block $block      Block instance.
+ * @return string Return the post comment's avatar.
+ */
+function render_block_core_post_comment_author_avatar( $attributes, $content, $block ) {
+	if ( ! isset( $block->context['commentId'] ) ) {
+		return '';
+	}
+
+	$comment = get_comment( $block->context['commentId'] );
+	if ( ! $comment ) {
+		return '';
+	}
+
+	// This is the only way to retreive style and classes on different instances.
+	$wrapper_attributes = WP_Block_Supports::get_instance()->apply_block_supports();
+	$width              = isset( $attributes['width'] ) ? $attributes['width'] : '96';
+	$height             = isset( $attributes['height'] ) ? $attributes['height'] : '96';
```

Comment:
Nitpick: `get_avatar` expects int values.

```suggestion
	$width              = isset( $attributes['width'] ) ? $attributes['width'] : 96;
	$height             = isset( $attributes['height'] ) ? $attributes['height'] : 96;
```

See https://github.com/WordPress/wordpress-develop/blob/4aed27209750cca396241470085eda0dc295f6f0/src/wp-includes/pluggable.php#L2646-L2647.

Reply:
Done!